### ISSUE TITLE
Snippet files created in wrong folder using UltiSnipsEdit

### ISSUE THREAD

**Issue by Alex-Canales** [SirVer/ultisnips#709]
Let say I am working in the folder `/home/me/myProject`, my vim configuration folder is `/home/me/.vim`. I am currently working on a JavaScript file.

If I have an already `/home/me/.vim/UltiSnips/javascript.snippets` file and I do a `:UltiSnipsEdit`:  I open and edit `/home/me/.vim/UltiSnips/javascript.snippets`.

If I do not have a `/home/me/.vim/UltiSnips/javascript.snippets` file and I do a `:UltiSnipsEdit`:  I open and edit `/home/me/myProject/UltiSnips/javascript.snippets`.

In a nutshell: `UltiSnipsEdit` creates a snippet in the local folder, instead of in the configuration folder, if no snippet file was already present in the configuration folder. Also, the snippets in this local file are not recognized.

Tested on vim 704

**Comment by seletskiy** [comment]
Related to #711.

**Comment by JikkuJose** [comment]
I am also experiencing the same. But followed the discussion in #711 & got a work around. Try adding this to your vimrc:

`let g:UltiSnipsSnippetDirectories = ['~/.vim/UltiSnips', 'UltiSnips']`

**Comment by Sathors** [comment]
Indeed, the workaround is working all right.

**Comment by SirVer** [comment]
Closing as dupe, consolidating this issue in #711.
